book.tex

```diff
@@ -17,7 +17,7 @@
 
 %\usepackage{amsmath}
 \usepackage{amsthm}
-\newtheorem{theorem}{Theorem}
+\newtheorem*{theorem}{Theorem}
```

Comment:
Is the asterisk the only difference? Does this remove theorem number somehow? Does it leave the rest of the theorem attributes in place?